src/components/PivotTable/PivotTableCell/PivotTableCell.test.tsx

```diff
@@ -0,0 +1,205 @@
+import { PivotTableCellTypes } from './model'
+
+const createComponent = ({ total, suffix }: PivotTableContextType, props: PivotTableCellProps) => {
+  return (
+    <PivotTableProvider value={{ total, suffix }}>
+      <PivotTableCell {...props} />
+    </PivotTableProvider>
+  )
+}
+
+describe('PivotTableCell', () => {
+  const [rowStart, columnStart] = [0, 0]
+  const gridArea = new GridArea(rowStart, columnStart)
+  const total = 1
+  const children = 1
+
+  it('should render correctly', () => {
+    const types = ['value'] as PivotTableCellTypes
+    const { container } = render(createComponent({ total }, { gridArea, types, children }))
+    expect(container).toMatchSnapshot()
+  })
+
+  it('should color font and background correctly', () => {
+    const types = ['value'] as PivotTableCellTypes
+    const expectedColor = 'rgb(0, 1, 2)'
+    const expectedBackgroundColor = 'rgb(2, 1, 0)'
+
+    const mockCalculateCellColor = jest.spyOn(utilsModule, 'calculateCellColor').mockReturnValue({
+      color: expectedColor,
+      backgroundColor: expectedBackgroundColor,
+    })
+
+    const { container } = render(createComponent({ total }, { gridArea, types, children }))
+    const { color, backgroundColor } = window.getComputedStyle(
+      selectPivotTableCellElement(container, rowStart, columnStart)
+    )
+
+    expect(color).toEqual(expectedColor)
+    expect(backgroundColor).toEqual(expectedBackgroundColor)
+    expect(mockCalculateCellColor).toBeCalled()
+  })
+
+  describe('hover', () => {
+    describe.each(['value', 'empty'])("when cell type includes '%s'", (typeValue) => {
+      const types = [typeValue] as PivotTableCellTypes
+
+      it('should update bg color on mouse enter', () => {
+        const { container } = render(createComponent({ total }, { gridArea, types, children }))
+        const cell = selectPivotTableCellElement(container, rowStart, columnStart)
+
+        const { backgroundColor: bgColorBeforeMouseEnter } = window.getComputedStyle(cell)
+        fireEvent.mouseEnter(cell)
+        const { backgroundColor: bgColorAfterMouseEnter } = window.getComputedStyle(cell)
+
+        expect(bgColorAfterMouseEnter).not.toEqual(bgColorBeforeMouseEnter)
+      })
+
+      it('should return to original bg color on mouse leave', () => {
+        const { container } = render(createComponent({ total }, { gridArea, types, children }))
+        const cell = selectPivotTableCellElement(container, rowStart, columnStart)
+
+        const { backgroundColor: initialBgColor } = window.getComputedStyle(cell)
+        fireEvent.mouseEnter(cell)
+        fireEvent.mouseLeave(cell)
+        const { backgroundColor: finalBgColor } = window.getComputedStyle(cell)
+
+        expect(initialBgColor).toEqual(finalBgColor)
+      })
+    })
+
+    it("when cell types dont include 'value' or 'empty', shouldnt update bg color on mouse enter", () => {
+      const types = ['header'] as PivotTableCellTypes
```

Comment:
```suggestion
      const typesWithHeader = ['header'] as PivotTableCellTypes
```
What do you think about this ?